girder/constants.py

```diff
@@ -128,6 +128,10 @@ class SettingKey:
     EMAIL_HOST = 'core.email_host'
     REGISTRATION_POLICY = 'core.registration_policy'
     SMTP_HOST = 'core.smtp_host'
+    SMTP_PORT = 'core.smtp.port'
```

Comment:
If someone had an existing setting for SMTP_HOST that contained the port, would this change break email for them?